params/superchain.go

```diff
@@ -37,6 +39,8 @@ var (
 	chaosnetRegolithTime = big.NewInt(1692156862)
 )
 
+var OPStackSupport = ProtocolVersionV0{Build: [8]byte{}, Major: 3, Minor: 1, Patch: 0, PreRelease: 1}.Encode()
```

Comment:
```suggestion
var OPStackSupport = ProtocolVersionV0{Build: [8]byte{}, Major: 6, Minor: 0, Patch: 0, PreRelease: 0}.Encode()
```
https://github.com/ethereum-optimism/op-geth/pull/243/files#diff-7663c343438d424a4d4046a455bf01862dea827d55849b3ac95a388c57c3899cR14

Current op-erigon supports Ecotone. need to update protocol version.

Reply:
Thanks, I updated it to 6.0.0 :+1: